src/main/resources/templates/index.twig

```diff
@@ -4,7 +4,7 @@
 <div style="height: 100vh">
     <div class="flex-center flex-column">
         <button type="button" class="btn btn-primary">I'm a Student</button>
-		<button type="button" class="btn btn-primary">I'm a Teacher</button>
+		<button type="button" onclick="location.href='/instructorViews'" class="btn btn-primary">I'm a Teacher</button>
```

Comment:
urls should be all lowercase

EDIT: I realize this is just my convention, sorry! We should discuss this and follow a convention

Reply:
I agree. I also think that we should just rename it to `instructor`.